### DISCUSSION TITLE
Additional maintainers or move to new GH Organization

### DISCUSSION THREAD

**Discussion by pschaeflein** [Huachao/vscode-restclient#826] [General]
Updates to this fantastic tool appear to have stopped.

No judgement - open source is not a job.

I am willing to help out. Is there a plan to add maintainers to the project? 

Is there community interest in copying the code to a new GitHub organization? 

I'm just looking for a way to get fixes/updates added to the project. At the time of this note, there are 21 PRs open, and I have a couple that I didn't submit yet.

**Comment by AnWeber** [comment]
I would also be willing to help. But I don't think a fork is the way to go. One main feature I need would be CLI support. This is currently difficult to implement. Other goals would be test support and easier extensibility of the api.
For this reason I started to implement my own application [httpyac](https://github.com/AnWeber/httpyac) ([vscode](https://marketplace.visualstudio.com/items?itemName=anweber.vscode-httpyac)). This can already do most of the functions like Rest Client and has the possibility to extend the program with NodeJS scripts. As next steps I would implement a plugin system similar to @vue/cli. The goal should be to not need pull requests into the main project most of the time, as these will always overtax individuals time wise.
Maybe this is interesting for you and you can take a look at it.

I hope the reimplementation is not perceived negatively, it was rather an attempt to solve my problems with vscode-restclient. I think the program and the idea behind it is great.

**Reply by lee-baker** [reply]
I had a look @AnWeber, looks really interesting. Going to install it tomorrow and give it a go. Changing from one big settings file to lots of .env files will be a challenge though.

**Reply by AnWeber** [reply]
@lee-baker Why do you need change from an big settings file to lots of .env files? I want to be compatible with rest-client if it is possible and may change my code if needed.

**Reply by lee-baker** [reply]
@AnWeber, how we use the rest client is to make the same api calls to all tenants of our application with different urls, authentication etc. So each tenant is a section in the settings file, which makes each a different environment. From what I can see, to achieve this in your current project I would need a .env file for each tenant.

**Reply by AnWeber** [reply]
@lee-baker I didn't quite understand the use case. If you are referring to the `rest-client.environmentVariables` setting, this corresponds to the `httpyac.environmentVariables` [setting](https://github.com/AnWeber/vscode-httpyac#environment-settings). I believe this is currently the only way to define environments in rest-client at the moment. I just extended it with the possibility of environments in dotenv (like #710 or #740) or [Intellij Format](https://www.jetbrains.com/help/idea/exploring-http-syntax.html#environment-variables)
Maybe you could show me a simplified example for your use case

**Comment by SjaaLom** [comment]
In my opinion the project could do with one or two more maintainers.

The community has taken the time to add to this great tool. It would be a shame to let these efforts go to waste.

As OP noted, open source is not a job, so no criticism here as well. 
But please consider adding another maintainer, because that can drastically increase the development speed of the tool.

**Comment by lee-baker** [comment]
@pschaeflein I would be happy to help make a github organisation. I am not experienced in this however I am willing to learn.

**Reply by pschaeflein** [reply]
Thanks. But the issue is not creating an organisation. Rather, getting buy-in from the original author and getting the ability to update the original entry in the VS Marketplace.

It has been over two months tho, so I might just start something new and credit/reference the original...

**Reply by lee-baker** [reply]
You are right, being able to update the original entry in the VS marketplace would be ideal. If we can't get @Huachao to respond we could always fork this and create a new vscode marketplace entry. Its not ideal but it might be what needs to happen to keep this alive. I am interested in being part of a team that does that.
